x-pack/plugins/security_solution/cypress/tasks/edit_rule.ts

```diff
@@ -8,7 +8,7 @@
 import { BACK_TO_RULE_DETAILS, EDIT_SUBMIT_BUTTON } from '../screens/edit_rule';
 
 export const saveEditedRule = () => {
-  cy.get(EDIT_SUBMIT_BUTTON).should('exist').click({ force: true });
+  cy.get(EDIT_SUBMIT_BUTTON).should('exist').click();
```

Comment:
Not sure how this change is related to the PR's objective. I'm afraid this could introduce some flakiness to Cypress security-tests, is it really needed?

Reply:
IDK @patrykkopycinski pushed that commit :)

Reply:
@patrykkopycinski Can you explain why this change was introduced? As I mentioned above, I'm unsure this can increase flakyness in our Cypress tests

Reply:
This PR was failing due to some flaky security_solutions test, he pushed these to fix them.
If this is not the case any more i can revert them.